include/arch/arm/aarch32/thread.h

```diff
@@ -121,6 +121,8 @@ struct _thread_arch {
 
 #if defined(CONFIG_USERSPACE)
 	uint32_t priv_stack_start;
+	uint32_t priv_stack_end;
```

Comment:
Might have missed that, but it looked to me these are only used in Cortex-R, so could we #ifdef them out from cortex-M, here?